app/src/main/java/com/example/roomremix2/data/FloorPlanData.kt

```diff
@@ -3,7 +3,11 @@ import android.widget.Toast
 import com.google.gson.Gson
 import java.io.File
 import java.io.FileOutputStream
-
+/**
+@property id; id of floorplan
+@property x; input x for room size.
+@property y; input y for room size.
```

Comment:
- Use proper KDoc format.
- What are property id, x, and y?  
- KDoc only adds value when it describes something over and above what can be read from code.